app/assets/stylesheets/form.scss

```diff
@@ -0,0 +1,4 @@
+.error {
+  border-color: #cc4c3f;
+  background-color: #FFE6E6;
```

Comment:
Color literals like `#FFE6E6` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFE6E6` should be written as `#ffe6e6`